### ISSUE TITLE
"Show End Slide" gives two different results when clicked back to back

### ISSUE THREAD

**Issue by astrodavid10** [WorldWideTelescope/wwt-windows-client#172]
Using animation for a picture, while moving the background sky and advancing time (viewing from a location in Sky mode), seems to cause the end slide to recognize two separate ending points. Clicking "Show End Slide" once provides one, and a second time, the other. I *think* that this may have to do with advancing time while moving the sky. Example tour attached.https://guides.github.com/features/mastering-markdown/
[animation_orion.zip](https://github.com/WorldWideTelescope/wwt-windows-client/files/5744164/animation_orion.zip)

**Comment by astrodavid10** [comment]
Not using animations while time advances seems to bypass the issue.

**Comment by pkgw** [comment]
Thanks for the report! I'm guessing that maybe there is a bug in how the frame progression state is synchronized between the main cameras and the animated items. For what it's worth, I suspect that the actual tour file should reliably play back as intended ... this is probably just an issue with how things are displayed in the editor, not the actual tour content.

**Comment by astrodavid10** [comment]
Yes, with further testing, I can confirm that the tour plays as intended, despite confusion within the editor.
